docs/xocto/localtime.md

```diff
@@ -19,6 +19,69 @@ from xocto import localtime
 
 See [xocto.localtime](https://github.com/octoenergy/xocto/blob/master/xocto/localtime.py) for more details, including examples and in depth technical details.
 
+## Localising datetimes between different timezones
+
+`localtime` can be used to change the timezone of a datetime e.g.
+
+```python
+london_tz = zoneinfo.ZoneInfo("Europe/London")
+dt = datetime.datetime(2024, 1, 1, tzinfo=london_tz)
+
+sydney_tz = zoneinfo.ZoneInfo("Australia/Sydney")
+localtime.as_localtime(dt, tz=sydney_tz)
+=> datetime.datetime(2024, 1, 1, 11, 0, tzinfo=zoneinfo.ZoneInfo(key='Australia/Sydney'))
+
+localtime.as_utc(dt)
+=> datetime.datetime(2024, 1, 1, 0, 0, tzinfo=zoneinfo.ZoneInfo(key='UTC'))
+```
+
+Behaviour around Daylight Savings Time (DST) boundaries - where
+clocks go forward or back - can be surprising e.g.
+
+```python
+# Clocks in GBR went forward on this day at 1AM => 2AM;
+# only one hour elapsed for the rest of the world.
+localtime.as_utc(datetime(2020, 3, 29, hour=2, tzinfo=london_tz)) -
+localtime.as_utc(datetime(2020, 3, 29, hour=0, tzinfo=london_tz))
+=> timedelta(seconds=3600)
```

Comment:
Is this right? This is showing 1 hour

Reply:
The example is accurate, yes:

- 2AM GBR => 1AM UTC
- 0AM GBR => 0AM UTC

I've elaborated the comment a bit; does that help?